marklogic-data-hub-central/ui/src/components/modeling/graph-view/side-panel/side-panel.tsx

```diff
@@ -224,6 +238,57 @@ const GraphViewSidePanel: React.FC<Props> = (props) => {
     }
   }, [modelingOptions.selectedEntity]);
 
+  useEffect(() => {
+    let loaded = true;
+    (async () => {
+      try {
+        const modelsResponse = await axios.get(`/api/models`);
+        const parsedModelData = entityFromJSON(modelsResponse.data);
+        const parsedEntityDef = entityParser(parsedModelData).filter(entity => entity.name && entity);
+
+        if (loaded) {
+          setEntityDefinitionsArray(parsedEntityDef);
+        }
+      } catch (error) {
+        handleError(error);
+      }
+    })();
+    return () => {
+      loaded = false;
+    };
+  }, [props.hubCentralConfig]);
+
+  const getEntityModels = async () => {
```

Comment:
Same comment here. Even if we can't use the Model call in browse component but i think we can placer this getEntityModels functions at a common location and refer here, since it seems to be performing the same task as in the entity display settings modal component earlier.